iceoryx_posh/include/iceoryx_posh/popo/untyped_publisher.hpp

```diff
@@ -75,6 +81,9 @@ class UntypedPublisherImpl : public base_publisher_t
 
 using UntypedPublisher = UntypedPublisherImpl<>;
 
+template <typename H>
+using UntypedPublisherWithCustomHeader = UntypedPublisherImpl<H>;
```

Comment:
Not sure if this is the best approach or if it's better to just have a
```
template <typename H>
using UntypedPublisher = UntypedPublisherImpl<H>;
```

Nevertheless, I think it might be a good idea to have something similar in the typed API to prevent the user from setting a different port.

Reply:
I'd keep `UntypedPublisher` the template param will default in most cases.

Reply:
That was also my first approach but then one has to use `UntypedPublisher<>` which might be cumbersome. But I guess it's a matter of taste and I will do what the majority wants

Reply:
We do not want to use ``UntypedPublisher<>`` which is the case that will occur mor often (most likely). So We need an alias for the custom header case. As for how it is to be named I have better proposal, and in doubt name it as it is (i.e. as you have).

First principles @elBoberido ,  simplicity:  "Of each particular thing, always ask what is it in itself."

So it is a HeadlessPublisher ...

Reply:
@MatthiasKillat I'm desperately waiting for better names. Both the Headless and Headed Untyped Publisher should be related with from the names

Reply:
Regarding naming, I don't have a better idea other than `UntypedPublisherWithCustomHeader`. That's what's inside, so it's fine.

@elBoberido Just realized, we need an example for the custom header feature! Have you planned to write one till Thursday?

Reply:
@mossmaurice not really, but I talked with @budrus and we decided to add it after the RC. Important would be to have the stable API until the release

Reply:
Kkay, I think the naming is the last blocker for the merge. Please make proposals

Reply:
I have none, CustomPublisher e.g. would not be sufficient to determine what can be customized (but be significantly shorter). The fact that it is untyped would also be implicit from use. I would say keep it like this for now.